### ISSUE TITLE
brew install icu4c: ModuleNotFoundError: No module named 'distutils.sysconfig'

### ISSUE THREAD

**Issue by timotheecour** [Homebrew/linuxbrew-core#12680]
- [x] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Homebrew/linuxbrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
https://gist.github.com/ded90a07dfff821d6162878080a61efc
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?
NA

To help us debug your issue please explain:
- What you were trying to do (and why)
brew install icu4c
- What happened (include command output)
- What you expected to happen
- Step-by-step reproduction instructions (by running `brew install` commands)

```
brew install icu4c
==> Downloading https://ssl.icu-project.org/files/icu4c/64.2/icu4c-64_2-src.tgz
Already downloaded: /home/ubuntu/.cache/Homebrew/downloads/19929a5617a0f24f6586b619c12a5b3c16274027374c69acdf978e7dfb0cabca--icu4c-64_2-src.tgz
==> ./configure --prefix=/home/linuxbrew/.linuxbrew/Cellar/icu4c/64.2 --disable-samples --disable-tests --enable-static
Last 15 lines from /home/ubuntu/.cache/Homebrew/Logs/icu4c/01.configure:
config.status: creating samples/cal/Makefile
config.status: creating samples/layout/Makefile
Not rebuilding data/rules.mk, assuming prebuilt data in data/in
Spawning Python to generate test/testdata/rules.mk...
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmp/icu4c-20190419-15425-1jqr1sr/icu/source/data/buildtool/__main__.py", line 19, in <module>
    import BUILDRULES
  File "/tmp/icu4c-20190419-15425-1jqr1sr/icu/source/test/testdata/BUILDRULES.py", line 4, in <module>
    from distutils.sysconfig import parse_makefile
ModuleNotFoundError: No module named 'distutils.sysconfig'
configure: error: Python failed to run; see above error.

READ THIS: https://docs.brew.sh/Troubleshooting

These open issues may also help:
icu4c: Build a bottle for Linuxbrew https://github.com/Homebrew/linuxbrew-core/pull/12658
```

**Comment by timotheecour** [comment]
workaround:
`sudo apt install python3-distutils`

**Comment by iMichka** [comment]
I built the bottle without issue in #12658.

> /usr/lib/python3.6/runpy.py

It looks like it picks up your system python? It should not.
I guess that we need to add `depends_on "python"` to the formula?

**Comment by iMichka** [comment]
This is probably fixed by #12658
